DDDetectors/src/Beampipe_o1_v01_geo.cpp

```diff
@@ -111,7 +111,7 @@ static dd4hep::Ref_t create_element(dd4hep::Detector& description,
   //Access to the XML File
   xml_det_t xmlBeampipe = element;
   const std::string name = xmlBeampipe.nameStr();
-
+  bool nocore  = xmlBeampipe.nocore(false);
```

Comment:
Yes, the idea is to put several BP detectors, the inner one (e.g. coating) with nocore=false and then several ones around it w/o core, like Be, water cooling tube, shields etc... I think that would be easier if I put it as a detector attribute, at least for the FCCee case. Your idea is to add an additional section attribute or to replace the detector attribute with a section one?